django/core/cache/backends/filebased.py

```diff
@@ -115,8 +115,12 @@ def clear(self):
     def _is_expired(self, f):
         """
         Take an open cache file `f` and delete it if it's expired.
+        An empty file `f` should be considered expired.
         """
-        exp = pickle.load(f)
+        try:
+            exp = pickle.load(f)
+        except EOFError:
+            exp = 0
```

Comment:
I think this would be better as an inline comment here: `# An empty file is considered expired.`